deb/ubuntu-cosmic/Dockerfile.aarch64

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:cosmic
+
+RUN apt-get update && apt-get install -y apparmor bash-completion libbtrfs-dev build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
```

Comment:
For maintainability, it would be good to put each package on a new line, and keep them sorted alphabetically. Doing so makes it easier to compare between Dockerfiles (and to do more granular review comments on GitHub);

something like;

```Dockerfile
RUN apt-get update
	&& apt-get install -y \
		apparmor \
		bash-completion \
		build-essential \
		ca-certificates \
		cmake curl \
		debhelper \
		dh-apparmor \
		dh-systemd \
		git \
		libapparmor-dev \
		libbtrfs-dev \
		libdevmapper-dev \
		libltdl-dev \
		libseccomp-dev \
		libsystemd-dev \
		pkg-config \
		vim-common \
	--no-install-recommends \
	&& rm -rf /var/lib/apt/lists/*
```